aggregation/aggregation_rule.go

```diff
@@ -36,6 +36,9 @@ func NewAggregationRule(aggSpec models.AggregationSpecification) Rule {
 	if aggSpec.Function == "" {
 		log.Fatalf("Rule %s must have a function", aggSpec.Name)
 	}
+	if len(aggSpec.GroupedDimensions) == 0 && len(aggSpec.Rollup.GroupedDimensions) != 0 {
```

Comment:
this doesn't handle the case where groupedDimensions is not 0 but also isn't a subset of rollup.groupedDimensions

Reply:
oh yes! I need to check that case too!! thanks!